ql/pom.xml

```diff
@@ -1105,7 +1105,6 @@
                   <include>org.apache.calcite.avatica:avatica</include>
                   <include>com.esri.geometry:esri-geometry-api</include>
                   <include>org.roaringbitmap:RoaringBitmap</include>
-                  <include>org.roaringbitmap:shims</include>
```

Comment:
Refer to https://github.com/RoaringBitmap/RoaringBitmap/pull/657/
Brief Description: shims project has been combined with RoaringBitmap project and shims project has been dropped completely.